cli/popper/commands/cmd_add.py

```diff
@@ -6,9 +6,10 @@
 import popper.utils as pu
 import shutil
 import yaml
-import zipfile
+import tarfile
 from io import BytesIO
 from popper.cli import pass_context
+import urllib
```

Comment:
seems like this is not needed anymore, right?

Reply:
Yeah. This will be removed before the final commit.